docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -36,22 +36,12 @@ Starting in version 7.14.0, {agent} is generally available (GA).
 
 The following steps will help you determine if {agent} can support your use case:
 
-. Determine if the integrations you need are supported and Generally Available (GA) on {agent}. Integrations that are currently GA are:
-+
-* `Auditd`
-* `Elastic Endpoint Security`
-* `Fortinet`
-* `Netflow`
-* `Office365`
-* `Okta`
-* `Palo Alto Networks`
-* `Suricata`
-* `System`
-* `Windows`
-* `Zeek`
-+
-NOTE: For more details, refer to {integrations-docs}[{integrations}].
-+
+. Determine if the integrations you need are supported and Generally Available
+(GA) on {agent}. To find out if an integration is GA, go to
+{integrations-docs}[{integrations}] and navigate to the docs for the
+integration. If there is no label, such as Beta or Experimental, the integration
+is GA.
```

Comment:
> If there is no label, such as Beta or Experimental, the integration
is GA.

This isn't the greatest user experience. If you give me a few days, I can have the overview table working again. That would be better to link to, I think.

Reply:
Agreed! That's awesome. I will wait.